core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/speech/TextToSpeech.java

```diff
@@ -69,7 +69,7 @@ private TextToSpeech() {
 	}
 
 	/**
-	 * @return TODO
+	 * @return Returns the static tts Object
 	 */
 	public static TextToSpeech getTTS() {
```

Comment:
Make TTS a Service and extract a simple interface

Reply:
Was genau bedeuted Service? Irgend etwas allgemeines oder gibt es das in unserem Projekt?

Reply:
https://github.com/AmyAssist/Amy/wiki/Dependency-Injection-(DI)#service